ibis/common/annotations.py

```diff
@@ -89,6 +96,17 @@ def optional(cls, validator=None, default=None):
             validator = option(validator, default=default)
         return cls(validator, default=None)
 
+    @classmethod
+    def varargs(cls, validator=None):
+        """Annotation to mark a variable length positional argument."""
+        validator = None if validator is None else tuple_of(validator)
+        return cls(validator, kind=VAR_POSITIONAL)
+
+    @classmethod
+    def varkwds(cls, validator=None):
```

Comment:
consider spelling it out as `varkeywords`

Reply:
How about `varkwargs` for consistency with the usual spelling of `*args, **kwargs`?

Reply:
Sounds good, updating.